start.sh

```diff
@@ -0,0 +1,3 @@
+/api/api &
```

Comment:
What if one process fails? We need to restart the container after that. Maybe there can be a better version of start.sh to monitor 2 processes and exit if one fails.

Reply:
yeah you're right, that can be done with `wait -n`